### PR TITLE
Feature/icons

### PR DESCRIPTION
**Pull request by percworld**
### What Changed?
- Creation of Slice for ChartView
- its reducer is now in store
- id renders both routes, ChartView and IncidentInfo.
- some styling changes (ongoing and past incidents scroll)
- 
### What's Next?
- It is time to paired-program again and swap off to Jeff for creation of the IncidentInfoSclice.js
- 
### Known Issues?
- None yet, we will need to get a request to begin functional testing.
- We are having difficulty again with the ongoing issue  rebasing based upon the all foder name change.  We will continue to look into this and moving ahead without a rebase on this merge into main.

### CODE REVIEW

File: src/components/OrgChart/OrgChart.js [hunk 1]
```diff
@@ -1,9 +1,92 @@
...
-  
+
+  const createChartData = () => {
+    //below data is for testing only (to be replaced with api call values)
+    const testObject = [
+      { id: 8, attributes: { name: 'chuck', title: 'PIO', email: 'chuck@chuck.com', phone: '123-123-123' } },
+      { id: 2, attributes: { name: 'chuck2', title: 'Safety Officer', email: 'chuck2@chuck.com', phone: '123-123-123' } },
+      { id: 5, attributes: { name: 'chuck5', title: 'Operations Chief', email: 'chuck5@chuck.com', phone: '123-123-123' } },
+      { id: 6, attributes: { name: 'chuck6', title: 'Finance Chief', email: 'chuck6@chuck.com', phone: '123-123-123' } }
+    ]
```
Comment (cbmackintosh) [inline src/components/OrgChart/OrgChart.js:14]: We can probably delete this test object right?

File: src/components/ChartView/ChartViewSlice.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      .addCase(getIncidentContacts.pending, state => {
+        state.status = 'loading'
+      })
+      .addCase(getIncidentContacts.fulfilled, (state, action) => {
+        state.status = 'idle'
+        state.incidentContacts = action.payload
+      })
+  }
+})
+
```
Comment (cbmackintosh) [inline src/components/ChartView/ChartViewSlice.js:31]: This setup should allow us to compile the chart using data from the server once that endpoint is up and running. Hopefully we'll need to make only minimal changes around how the data is structured.

File: src/API-calls.js [hunk 3]
```diff
@@ -48,4 +48,9 @@ export const runNameQuery = (queryString) => {
 export const fetchContactDets = (id) => {
   return fetch(`${baseURL}/contacts/${id}`)
     .then(checkResponse)
+}
+
+export const fetchIncidentContacts = (id) => {
+  return fetch(`${baseURL}/incidents/${id}/contacts`)
+    .then(checkResponse)
```
Comment (cbmackintosh) [inline src/API-calls.js:55]: Let's remember to ask Backend to move this endpoint up the priority list. We want to be able to build charts with this data over the weekend

**Review by JeffShepherd — APPROVED** [review]
Looks good! All updates made while pair programming.

**Review by cbmackintosh — APPROVED** [review]
This was a very productive paired programming session! There's still a lot of work to do, but I think we're starting to get a clear sense of how core functionality will be established. I'm confident that we can achieve this over the weekend!

**Pull request merged**